### DISCUSSION TITLE
Add Custom tag to improve request management

### DISCUSSION THREAD

**Discussion by Hug0lef** [sct/overseerr#2949] [Feature Requests]
Add a way to put tags others than "requested" or "approved" in order to simplify manual downloading process, like "airing" for shows that aren't already fully released or "unavailable"

**Comment by Drallas** [comment]
What you mean is the Status 'Tag' on requests?

There are also tags that can be set when you make a request. 
The values are pulled in from Sonarr / Radarr, but can't be set directly from Overseer.
This is also a bit confusing.

**Reply by Hug0lef** [reply]
I don't use Radarr or Sonarr, i get the content myself.

By tag i'm talking about what is written next to status : 
![image](https://user-images.githubusercontent.com/91483717/186636391-0a6c716f-c824-404f-ab90-96b4509f6a6e.png)
If i understood what you sayed, those are from sonarr and radarr, what i would like is to be able to add custom ones.
